<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>A CS123 Final</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="View" name="view" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>247</width>
     <height>38</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>247</width>
     <height>524287</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QSlider" name="sunPositionSlider">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sun Position</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sunPositionText">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="octavesSlider">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Terrain Octaves</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="octavesText">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="normalMapping">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Normal Map</string>
     </property>
    </widget>
    <widget class="QSlider" name="snowAmountSlider">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="snowAmountText">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>260</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>280</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Snowfall Amount</string>
     </property>
    </widget>
    <widget class="QSlider" name="positionSlider">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="positionText">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Camera Position</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="snowStatus">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Snow</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>View</class>
   <extends>QWidget</extends>
   <header>view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
